src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -749,6 +751,8 @@ public function getSqlType($type, $limit = null)
                 }
 
                 return ['name' => 'blob'];
+            case static::PHINX_TYPE_BIT:
+                return ['name' => 'bit', 'limit' => $limit ?: 64];
```

Comment:
Sorry for commenting again, but wouldn't it be better to use `1` instead `64` here? If you omit the size when you declare `bit` column in the MySQL, I think it will be `bit(1)`. Or would using maximum size is a usual thing in Phinx?

Reply:
Sorry, @chinpei215. Maybe I had a little misunderstanding here... But following STRING, CHAR, INTEGER definitions, I thoght this number was the maximum size (in bytes) associated with the type. 

And accordingly to MySQL documentation, BIT can vary from 1 to 64.

But I don't know now... What do you think?

And about file permissions (644). Is that right?

Reply:
OK. Then I am OK that `$limit` defaults to 64.
Regarding file permissions, I think you can confirm it when you click the `Changes` tab of this PR. I think you can see 100644 → 100755 after the each file names.

Reply:
FYI @chinpei215 @shbmira The options returned by `MySQLAdapter->getSqlType()` actually act as the default options.

This means:
1. `BIT` columns added by phinx will by default have a limit of 64, unless overridden by the developer.
2. A developer may still specify a limit > 64, but the migration will fail:
  `SQLSTATE[42000]: Syntax error or access violation: 1439 Display width out of range for column 'mybit' (max = 64)`